Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/StubGenerator.kt

```diff
@@ -848,6 +848,16 @@ class StubGenerator(
             }
         }
 
+        nativeIndex.wrappedMacros.forEach {
+            try {
+                stubs.add(
+                        GlobalVariableStub(GlobalDecl(it.name, it.type, isConst = true), this)
```

Comment:
This is ok, but `GlobalVariableStub` has special handling for struct- and array-typed C globals, and may generate incorrect stubs for such macros.
I suppose `GlobalVariableStub` could distinguish C macro and C global cases, and don't do special handling for the first one.